### PR TITLE
Futurize outliers ransac

### PR DESCRIPTION
**Pull request by arnaudbore**
@mdesco I would need you to test this script and I would need at least two reviewers to go through the code.

Thank you

### CODE REVIEW

File: scripts/scil_remove_outliers_ransac.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        parser.error('--min_fit should be at least 2. Current value: {}'
+                     .format(args.min_fit))
+    if args.max_iter < 1:
+        parser.error('--max_iter should be at least 1. Current value: {}'
+                     .format(args.max_iter))
+    if args.fit_thr <= 0:
+        parser.error('--fit_thr should be greater than 0. Current value: {}'
+                     .format(args.fit_thr))
+
+    logging.basicConfig(level=getattr(logging, args.log))
```
Comment (GuillaumeTh) [inline scripts/scil_remove_outliers_ransac.py]: `log` doesn't exist. Please fix.

**Comment by GuillaumeTh** [comment]
Can we merge ? Maybe an other approve before ? @boiteaclou ?

File: scripts/scil_remove_outliers_ransac.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                        'to fit the model. [%(default)s]')
+    p.add_argument('--max_iter', type=int, default=1000,
+                   help='The maximum number of iterations allowed ' +
+                        'in the algorithm. [%(default)s]')
+    p.add_argument('--fit_thr', type=float, default=1e-2,
+                   help='Threshold value for determining when a data ' +
+                        'point fits a model. [%(default)s]')
+
+    add_verbose_arg(p)
+
```
Comment (frheault) [inline scripts/scil_remove_outliers_ransac.py]: No newline

File: scripts/scil_remove_outliers_ransac.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Remove outliers from image using the RANSAC algorithm.
+The RANSAC algorithm parameters are sensitive to the input data.
+
+NOTE: Current default parameters are tuned for ad/md/rd images only.
```
Comment (frheault) [inline scripts/scil_remove_outliers_ransac.py]: Maybe a warning if the input image average signal (excluding 0) are above 0.01.

I cannot really make it work for a FA, so I would write that it is not recommended to use on something else.

File: scripts/scil_remove_outliers_ransac.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                     .format(args.max_iter))
+    if args.fit_thr <= 0:
+        parser.error('--fit_thr should be greater than 0. Current value: {}'
+                     .format(args.fit_thr))
+
+    if args.verbose:
+        logging.basicConfig(level=logging.DEBUG)
+
+    in_img = nib.load(args.in_image)
+    in_data = in_img.get_data()
```
Comment (jchoude) [inline scripts/scil_remove_outliers_ransac.py]: `get_fdata()`

File: scripts/scil_remove_outliers_ransac.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+    if np.average(in_data[in_data > 0]) > 0.1:
+        logging.warning('Be carefull, your image doesn\'t seem to be an ad, '
```
Comment (jchoude) [inline scripts/scil_remove_outliers_ransac.py]: `carefull` -> `careful`

**Pull request merged**